python/unique/yashagarwal1999_unique_array.py

```diff
@@ -0,0 +1,11 @@
+def getunique(arr):
+    finallist=[]
+    for i in arr:
+        if(i not in finallist):
+            finallist.append(i);
+    return finallist
```

Comment:
I'm wondering how this can be shrunk even further, perhaps even via a comprension.

Reply:
Solved the issue using list comprehension.
Please check